plugins/woocommerce-admin/client/marketplace/components/content/content.tsx

```diff
@@ -212,30 +340,46 @@ export default function Content(): JSX.Element {
 		recordLegacyTabView( marketplaceViewProps );
 	}, [ query?.tab, query?.term, query?.section, query?.category ] );
 
+	// Reset current page when tab, term, or category changes
+	useEffect( () => {
+		setCurrentPage( 1 );
+		setFirstNewProductId( 0 );
+	}, [ selectedTab, query?.category, query?.term ] );
+
+	const getProductType = ( tab: string ): ProductType => {
+		switch ( tab ) {
+			case 'themes':
+				return ProductType.theme;
+			case 'business-services':
+				return ProductType.businessService;
+			default:
+				return ProductType.extension;
+		}
+	};
```

Comment:
Should this go into `utils/functions.tsx`? Doesn't necessarily need to be done as part of this PR but I feel like we can improve the whole `ProductType` and string conversion flow

Reply:
On related note, [this](https://github.com/woocommerce/woocommerce/blob/554434ea3d366ce688c3d945c2e617cf83a32554/plugins/woocommerce-admin/client/marketplace/components/category-selector/category-selector.tsx#L20) feels weird. However, it'd require an API change as well, since these work:

- https://woocommerce.com/wp-json/wccom-extensions/1.0/search?country=MK (extensions)
- https://woocommerce.com/wp-json/wccom-extensions/1.0/search?category=themes&country=MK (themes)

But this doesn't:
- https://woocommerce.com/wp-json/wccom-extensions/1.0/search?category=extensions&country=MK (extensions)

Reply:
Can create an issue about it and unblock this PR.

Reply:
> Should this go into utils/functions.tsx? Doesn't necessarily need to be done as part of this PR but I feel like we can improve the whole ProductType and string conversion flow

It could, yeah. I had actually looked if I couldn't replace some of the other ProductType checks in the component with this function, but it doesn't look that way. Moved the function in aadaeb3

> On related note, [this](https://github.com/woocommerce/woocommerce/blob/554434ea3d366ce688c3d945c2e617cf83a32554/plugins/woocommerce-admin/client/marketplace/components/category-selector/category-selector.tsx#L20) feels weird. However, it'd require an API change as well, since these work:

Yeah, I suspect originally this was designed to only support extensions, and then themes were added afterwards. To fix on the API side will likely mean we need to support two versions of the endpoint for backwards compatibility.